apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.thumbnail.presenter.tsx

```diff
@@ -4,33 +4,38 @@ import {
     ResourceType,
 } from '@coscrad/api-interfaces';
 import { MediaPlayer } from '@coscrad/media-player';
-import { Link } from 'react-router-dom';
+import { Card, CardContent } from '@mui/material';
 import { routes } from '../../../app/routes/routes';
 import { FloatSpacerDiv, SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceNavLink } from '../shared/resource-nav-link';
 import { convertMillisecondsToSeconds } from '../utils/math/';
-import './transcribed-audio-detail.thumbnail.presenter.css';
+import styles from './transcribed-audio-detail.thumbnail.presenter.module.scss';
 
 export const TranscribedAudioDetailThumbnailPresenter = ({
     id,
     lengthMilliseconds,
     audioURL,
 }: ICategorizableDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => (
-    <div className="detail-thumbnail-container">
-        <Link to={`/${routes.resources.ofType(ResourceType.transcribedAudio).detail(id)}`}>
-            <div
-                className="detail-thumbnail-meta-container"
-                title="View Connected Transcribed Audio"
-            >
+    <Card>
+        <CardContent>
+            <div data-testid={id} className={styles['preview']}>
+                <MediaPlayer audioUrl={audioURL} />
```

Comment:
We don't want media players in the thumbnail presenters. Instead, we should do the ["Spotify experience" story](https://www.pivotaltracker.com/story/show/184048108) when playing audio in a multiple categorizables panel becomes important.